dist/platforms/knex/0.16/index.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+
+var _index = require('../0.12/index');
```

Comment:
not sure if 0.12 was intentional or not

Reply:
@MarkHerhold yes mocking the knex library hasn't changed since `0.12`